.github/workflows/ci.yml

```diff
@@ -58,6 +58,10 @@ jobs:
           git --version
           node --version
           npm --version
+      - name: Install commitlint
+        run: npm install
+      - name: Print commitlint version
+        run: npx commitlint --version
```

Comment:
@realmarv this change seems redundant because commitlint.sh (which is called in the next step) already calls these commands above

Reply:
Done